infrastructure/test_config.tf

```diff
@@ -0,0 +1 @@
+
```

Comment:
Is this file needed?

Reply:
No idea. Probably not. I find this file in almost all the repositories. So just left it l;ike that. But if not needed we can remove it .

Reply:
not needed